unicoremc/models.py

```diff
@@ -270,6 +259,24 @@ def to_dict(self):
             'docker_cmd': self.docker_cmd or '',
         }
 
+    def get_website_manager(self):
+        if not (self.application_type and self.application_type.project_type):
+            raise exceptions.ProjectTypeRequiredException(
+                'project_type is required')
+
+        # For non-standalone apps, we use the IogtWebsiteManager
+        if not self.own_repo():
+            return IogtWebsiteManager(self)
```

Comment:
@miltontony I'm wrong about that failure :) The Iogt manager is being returned here :) But it will also be returned for springboard and unicore-cms apps. Perhaps the workflow classes can be broken down into more generic workflows that can be composed for different app builds?